CMakeLists.txt

```diff
@@ -237,10 +274,12 @@ install(FILES
 #=============================================
 # Add uninstall target
 #=============================================
-configure_file(
-	"${CMAKE_CURRENT_SOURCE_DIR}/cmake_uninstall.cmake.in"
-	"${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake"
-	IMMEDIATE @ONLY)
-
-add_custom_target(uninstall
-	COMMAND ${CMAKE_COMMAND} -P ${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake)
+if (NOT TARGET uninstall)
```

Comment:
Can you update the README to reflect this change?

Reply:
ok